docs/setup.md

```diff
@@ -80,12 +80,41 @@ You will need to configure some listeners to make the bridge functional.
 ```
 
 At a minimum, you should bind the `webhooks` resource to a port and address. You can have multiple resources on the same
-port, or one on each.
+port, or one on each. Each listener MUST listen on a unique port.
 
 You will also need to make this port accessible to the internet so services like GitHub can reach the bridge. It
 is recommended to factor hookshot into your load balancer configuration, but currrently this process is left as an
 excercise to the user.
 
+In terms of API endpoints:
+
+- The `webhooks` resource handles resources under `/`, so it should be on it's own listener.
```

Comment:
> should be on its on listener

but you can reasonably assume that webhooks will not get assigned an endpoint `/metrics` or `/v1...`.... right?!